libraries/AP_NavEKF3/AP_NavEKF3_MagFusion.cpp

```diff
@@ -1427,7 +1427,7 @@ void NavEKF3_core::alignMagStateDeclination()
 
     // rotate the NE values so that the declination matches the published value
     Vector3F initMagNED = stateStruct.earth_magfield;
-    ftype magLengthNE = norm(initMagNED.x,initMagNED.y);
+    ftype magLengthNE = initMagNED.xy().length();
```

Comment:
I guess there's no difference between norm and length() even for ftype?

Reply:
I will plead a little ignorance here. I was surprised by the savings.
For this specific case initMagNED it is an fytpe so there is no difference. 

From what I can see in debugger and via inspection, T length(void) calls ftype norm(void) each function of  <template T> so there is output type narrowing that is possible if the inputs are floats and ftype -> is double?